src/pkg/data/consumers/governor/mock_cluster_client_api.go

```diff
@@ -0,0 +1,297 @@
+// Code generated by mockery v2.20.2. DO NOT EDIT.
```

Comment:
I wouldn't overcomplicate at this point. The updateTelemetry endpoint will hardly ever change so I think it's fine to generate mocks in case it changes and just add them in the PR.

We currently we are including go client model classes directly so I think it's fine to include mocks too. If at some point we import go client from somewhere, then generating mock clases would make more sense.

Reply:
ok, then we can go with out auto mock generation.